showcase/src/app/demo-ui/pages/generic-search/components/demo-generic-search-form.component.ts

```diff
@@ -24,7 +24,7 @@ export class DemoGenericSearchFormComponent implements OnInit, OnChanges, StarkS
 	public heroOptions: string[] = [];
 	public movieOptions: string[] = [];
 
-	public searchForm: FormGroup;
+	public searchForm: FormGroup = this.createSearchForm(this.searchCriteria);
```

Comment:
But this logic is just for the default value/initialization, if the the input is set there is already logic in ngOnChanges to handle the update. Otherwise I'm doing the same thing twice. 😕

Reply:
I was wrong for the initialization, but I refactored that does follow that logic.

Reply:
Add back the isEqual